examples/bootstrap.php

```diff
@@ -52,3 +54,15 @@ function logger(): LoggerInterface
 
     return new ConsoleLogger(new ConsoleOutput($verbosity));
 }
+
+function print_token_usage(Metadata $metadata): void
+{
+    $tokenUsage = $metadata->get('token_usage');
+
+    assert($tokenUsage instanceof TokenUsage);
+
+    echo 'Utilized Tokens: '.$tokenUsage->totalTokens.\PHP_EOL;
+    echo '-- Prompt Tokens: '.$tokenUsage->promptTokens.\PHP_EOL;
+    echo '-- Completion Tokens: '.$tokenUsage->completionTokens.\PHP_EOL;
+    echo 'Remaining Tokens: '.$tokenUsage->remainingTokens.\PHP_EOL;
```

Comment:
Let's print all tokens here which are not null as we have more properties in the TokenUsage object

Reply:
Done.